docs/user-guide/deployments/deploy-on-kubernetes/getting-started.md

```diff
@@ -0,0 +1,442 @@
+NAMESPACE: etcd-cluster
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+CHART NAME: etcd
+CHART VERSION: 10.2.12
+APP VERSION: 3.5.15
+
+** Please be patient while the chart is being deployed **
+
+etcd can be accessed via port 2379 on the following DNS name from within your cluster:
+
+    etcd.etcd-cluster.svc.cluster.local
+
+To create a pod that you can use as a etcd client run the following command:
+
+    kubectl run etcd-client --restart='Never' --image docker.io/bitnami/etcd:3.5.15-debian-12-r6 --env ETCDCTL_ENDPOINTS="etcd.etcd-cluster.svc.cluster.local:2379" --namespace etcd-cluster --command -- sleep infinity
+
+Then, you can set/get a key using the commands below:
+
+    kubectl exec --namespace etcd-cluster -it etcd-client -- bash
+    etcdctl  put /message Hello
+    etcdctl  get /message
+
+To connect to your etcd server from outside the cluster execute the following commands:
+
+    kubectl port-forward --namespace etcd-cluster svc/etcd 2379:2379 &
+    echo "etcd URL: http://127.0.0.1:2379"
+
+WARNING: There are "resources" sections in the chart not set. Using "resourcesPreset" is not recommended for production. For production installations, please set the following values according to your workload needs:
+- disasterRecovery.cronjob.resources
+- resources
+  +info https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+```
+</details>
+
+Wait for the etcd cluster to be ready:
+
+```bash
+kubectl get pods -n etcd-cluster -l app.kubernetes.io/instance=etcd
+```
+
+<details>
+  <summary>Expected Output</summary>
+```bash
+NAME     READY   STATUS    RESTARTS   AGE
+etcd-0   1/1     Running   0          2m8s
+etcd-1   1/1     Running   0          2m8s
+etcd-2   1/1     Running   0          2m8s
+```
+</details>
+
+You can test the etcd cluster by running the following command:
+
+```bash
+kubectl -n etcd-cluster \
+  exec etcd-0 -- etcdctl endpoint health \
+  --endpoints=http://etcd-0.etcd-headless.etcd-cluster.svc.cluster.local:2379,http://etcd-1.etcd-headless.etcd-cluster.svc.cluster.local:2379,http://etcd-2.etcd-headless.etcd-cluster.svc.cluster.local:2379
+```
+
+<details>
+  <summary>Expected Output</summary>
+```bash
+http://etcd-1.etcd-headless.etcd-cluster.svc.cluster.local:2379 is healthy: successfully committed proposal: took = 3.008575ms
+http://etcd-0.etcd-headless.etcd-cluster.svc.cluster.local:2379 is healthy: successfully committed proposal: took = 3.136576ms
+http://etcd-2.etcd-headless.etcd-cluster.svc.cluster.local:2379 is healthy: successfully committed proposal: took = 3.147702ms
+```
+</details>
+
+## Install the GreptimeDB cluster with monitoring integration
+
+Now that the GreptimeDB Operator and etcd cluster are installed, you can deploy a minimum GreptimeDB cluster with monitoring integration:
+
+:::warning
+The default configuration for the GreptimeDB cluster is not recommended for production use. 
+You should adjust the configuration according to your requirements.
```

Comment:
```suggestion
You should [adjust the configuration](greptimedb-operator-management.md#configuration) according to your requirements.
```

Reply:
It's the cluster configuration. The docs are still wip.